community/event_management/web_template/schedule_section/schedule_section.html

```diff
@@ -0,0 +1,74 @@
+{% set schedule_docs = frappe.get_all("Schedule", fields =["speaker", "slot"], filters={
+'event': event
+},) %}
+{% set slot_data = {} %}
+{% for schedule in schedule_docs %}
+{% set speaker = frappe.get_doc("Speaker Registration", schedule.speaker) %}
+{% set slot = frappe.get_doc("Slot", schedule.slot) %}
+<div class="hide">{{ slot_data.setdefault(slot.date, []).append({
```

Comment:
Why is this in a div?

Reply:
It prints None as a value, above the section. Using a div, so I could remove it from the DOM.

Reply:
You are just setting values in slot_data, right? 

`{% set slot_data ...... %}`

Shouldn't this be enough?

Reply:
Doesnot work for append function in jinja. I tried that. Was getting errors. So had to use {{ }}.

Reply:
Saw a different method for using append in jinja. That would element the problem of using div.